src/styles/styles.js

```diff
@@ -1256,6 +1267,18 @@ const styles = {
         fontSize: variables.iouAmountTextSize,
     }, 0),
 
+    iouPreviewBox: {
+        borderColor: themeColors.border,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
```

Comment:
Could we add a new variable, something like `variables.componentBorderRadiusCard` and set it to `12` and then use it here? I think we want all card-like components to have a slightly bigger border radius than normal.